dashboard/urls.py

```diff
@@ -100,6 +97,14 @@
         path('lti/jwks/', lti_new.get_jwks, name="get_jwks"),
     )
 
+if settings.ENABLE_BACKEND_LOGIN:
+    from django.contrib.auth import views as auth_views
+    # Login patterns for testing, SAML should be installed in prod
+    urlpatterns += (
+        path('accounts/login/', auth_views.LoginView.as_view(), name='login'),
+        path('accounts/logout/', auth_views.LogoutView.as_view(), name='logout'),
```

Comment:
This is a nice way to reduce the access to the login/logout URL. 

In `settings.py` I see as below 
```
# Add any settings you need to be available to templates in this array
SETTINGS_EXPORT = ['LOGIN_URL','LOGOUT_URL','DEBUG', 'GA_ID', 'RESOURCE_VALUES']
```

do we need to have a check if when either saml/lti check enabled and then add LOGIN_URL and LOGOUT_URL?

Reply:
The SETTINGS_EXPORT is for this package https://pypi.org/project/django-settings-export/

So those settings are available in the template. I think we always want some URL sent to be able to login logout. 

We may want the to disable the login/logout URL in the UI in some cases (like if LTI is enabled?) but that seems like a separate issue.

Reply:
It looks like #784 moved many of these out of the template to the view so I think you're right we don't need this package anymore. I'll create a new issue.

Reply:
Created as #1084